pkgs/development/python-modules/langchain-classic/default.nix

```diff
@@ -0,0 +1,171 @@
+    "langchain-core"
+    "numpy"
+    "tenacity"
+  ];
+
+  dependencies = [
+    aiohttp
+    langchain-core
+    langchain-text-splitters
+    langsmith
+    numpy
+    pydantic
+    pyyaml
+    requests
+    sqlalchemy
+    tenacity
+  ]
+  ++ lib.optional (pythonOlder "3.11") async-timeout;
+
+  optional-dependencies = {
+    numpy = [ numpy ];
+  };
+
+  nativeCheckInputs = [
+    blockbuster
+    freezegun
+    httpx
+    lark
+    pandas
+    pytest-asyncio
+    pytest-mock
+    pytest-socket
+    pytestCheckHook
+    requests-mock
+    responses
+    syrupy
+    toml
+  ];
+
+  pytestFlags = [
+    "--only-core"
+  ];
+
+  enabledTestPaths = [
+    # integration_tests require network access, database access and require `OPENAI_API_KEY`, etc.
+    "tests/unit_tests"
+  ];
+
+  disabledTests = [
+    # These tests have database access
+    "test_table_info"
+    "test_sql_database_run"
+    # These tests have network access
+    "test_socket_disabled"
+    "test_openai_agent_with_streaming"
+    "test_openai_agent_tools_agent"
+    # This test may require a specific version of langchain-community
+    "test_compatible_vectorstore_documentation"
+    # AssertionErrors
+    "test_callback_handlers"
+    "test_generic_fake_chat_model"
+    # Test is outdated
+    "test_serializable_mapping"
+    "test_person"
+    "test_aliases_hidden"
+    # ImportError: OpenAI is required for FlareChain. Please install langchain-openai.pip install langchain-openai
+    "test_from_llm_rejects_non_chatopenai"
+  ];
+
+  disabledTestPaths = [
+    # pydantic.errors.PydanticUserError: `ConversationSummaryMemory` is not fully defined; you should define `BaseCache`, then call `ConversationSummaryMemory.model_rebuild()`.
+    "tests/unit_tests/chains/test_conversation.py"
+    # pydantic.errors.PydanticUserError: `ConversationSummaryMemory` is not fully defined; you should define `BaseCache`, then call `ConversationSummaryMemory.model_rebuild()`.
+    "tests/unit_tests/chains/test_memory.py"
+    # pydantic.errors.PydanticUserError: `ConversationSummaryBufferMemory` is not fully defined; you should define `BaseCache`, then call `ConversationSummaryBufferMemory.model_rebuild()`.
+    "tests/unit_tests/chains/test_summary_buffer_memory.py"
+    "tests/unit_tests/output_parsers/test_fix.py"
+    "tests/unit_tests/chains/test_llm_checker.py"
+    # TypeError: Can't instantiate abstract class RunnableSerializable[RetryOutputParserRetryChainInput, str] without an implementation for abstract method 'invoke'
+    "tests/unit_tests/output_parsers/test_retry.py"
+    # pydantic.errors.PydanticUserError: `LLMSummarizationCheckerChain` is not fully defined; you should define `BaseCache`, then call `LLMSummarizationCheckerChain.model_rebuild()`.
+    "tests/unit_tests/chains/test_llm_summarization_checker.py"
+  ];
+
+  pythonImportsCheck = [ "langchain_classic" ];
+
+  passthru = {
+    skipBulkUpdate = true;
+    updateScript = gitUpdater {
```

Comment:
Again, is there a reason `gitUpdater` is used instead of `nix-update-script { extraArgs = [ "--version-regex 'langchain==(.*)'" ]; }`?